main/http-definitions.bs

```diff
@@ -0,0 +1,27 @@
+HTTP Definitions {#http-definitions}
+===================================
+
+## Link Relations ## {#link-relations}
+
+The intent is that these link relations will be registered with IANA per [[!RFC8288]]. 
+
+### acl ### {#acl}
+
+The contents of this section were originally taken from [Web Access Control](https://www.w3.org/wiki/WebAccessControl).
+
+The following Link Relationship will be submitted to IANA for review, approval, and inclusion in the IANA Link Relations registry.
+
+: Relation Name
+:: `acl`
+: Description
+:: The relationship `A acl B` asserts that resource B provides access control description of resource A. There are no constraints on the format or representation of A. B MUST be a Access Control List document using a Resource Description Framework syntax.
+: Reference
+:: This specification.
+: Notes
+:: Consumers of ACL resources should be aware of the source and chain of custody of the data.
+
```

Comment:
what is "the chain of custody of the data"?

Reply:
Authority. Just an initial note. If it is not significant we can remove it. The note can be empty or anything ( https://www.iana.org/assignments/link-relations/link-relations.xhtml )

While Solid currently expects both the resource and its ACL on the same server, it has applicability beyond Solid - which is why we are considering it to push it through IANA. So, technically possible to have A and B on different servers and authority. I don't suggest that restriction is placed on the definition of the relation ie. okay to keep it only for Solid for now.

Reply:
I think it makes sense to have the ACL somewhere else. They could be on a future hyper-blockchain build on linked data for example.

Reply:
Anyway, I think the phrasing could be better. But presumably this will be reviewed a lot more before sending it to IANA.

Reply:
In addition to more reviewing, it needs to refer to an actual spec - so a version of the Solid spec that can be acknowledged.